example/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   cke:
     container_name: cke
-    image: quay.io/cybozu/cke:1.20.0
+    image: quay.io/cybozu/cke:1.21.0-rc.1
```

Comment:
1.21.0-rc. **2** ?

Reply:
All right. I don't request change at this line.

Reply:
The compose test has failed for 1.21.0-rc.1...
I'll check the logs of the release workflow for 1.21.0-rc1.